src/libraries/Microsoft.Extensions.FileProviders.Physical/tests/PhysicalFileProviderTests.cs

```diff
@@ -92,7 +92,6 @@ public void GetFileInfoReturnsNotFoundFileInfoForIllegalPathWithLeadingSlashes_U
 
         [Fact]
         [PlatformSpecific(TestPlatforms.Linux)]
-        [OuterLoop]
```

Comment:
Any idea why this test was marked as OuterLoop in the first place? Or asking a different way: Why did you decide to remove it?

Reply:
I added it, then marked it that way because I wasn't sure if the issue was due to system load.